pygmt/base_plotting.py

```diff
@@ -606,47 +606,52 @@ def grdview(self, grid, **kwargs):
 
     @fmt_docstring
     @use_alias(
-        R="region",
-        J="projection",
+        A="straight_lines",
```

Comment:
`straight_lines` or `straight_line`?

Reply:
GMT.jl uses `straight_lines`. But maybe we should use `straight_line` here to keep it consistently singular.

Reply:
Yes to me.